decidim-core/spec/services/decidim/html_truncation_spec.rb

```diff
@@ -0,0 +1,177 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe Decidim::HtmlTruncation do
+  let(:subject) { described_class.new(text, options).perform }
+  let(:options) do
+    {
+      max_length: max_length,
+      tail: tail,
+      count_tags: count_tags,
+      count_tail: count_tail,
+      tail_before_final_tag: tail_before_final_tag
+    }
+  end
+  let(:max_length) { 30 }
+  let(:tail) { "..." }
+  let(:count_tags) { false }
+  let(:count_tail) { false }
+  let(:tail_before_final_tag) { false }
+  let(:text) { ::Faker::Lorem.paragraph(sentence_count: 25) }
+
+  describe "empty content" do
+    let(:text) { "" }
+
+    it "does not do anything" do
+      expect(subject).to eq(text)
+    end
+  end
+
+  describe "short content" do
+    let(:max_length) { 100 }
+    let(:texts) do
+      [
+        "Mauris sed libero.",
+        'foo <a href="www.example.com">link</a> bar',
+        "some <strong>text</strong> here",
+        "<b><em>foo</em></b>"
+      ]
+    end
+
+    it "does not get cutted" do
```

Comment:
```suggestion
    it "does not get cut" do
```
I think "cutted" is not correct in English
https://www.italki.com/post/question-385836?hl=es